### PR TITLE
Connector Health: Fixed SAT for Instagram, Facebook pages, Google Search Console, Stripe, Typeform, Hubspot, Sentry, Faker, Zendesk support, Slack, Jira

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
Fixed SAT for Instagram, Facebook pages, Google Search Console, Stripe, Typeform, Faker, Zendesk support, Slack, Jira

## How
Updated expected records. For Facebook pages updated ignored field in full refresh test to new format.

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml [hunk 1]
```diff
@@ -38,11 +44,3 @@ acceptance_tests:
         timeout_seconds: 1800
         future_state:
           future_state_path: "integration_tests/abnormal_state.json"
-        cursor_paths:
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml:41]: is this change intentional? what's the reason?

Comment (lazebnyi) [reply airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml:41]: No. Cursor fields already present in `configured_catalog_incremental.json`.

**Review by midavadim — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml (hunk 1 above)
Comment (davydov-d) [reply airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml:41]: Those are different. Here we should describe a path to a cursor value in a state, not in a record. Like [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-search-console/integration_tests/abnormal_state.json) and [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-search-console/source_google_search_console/streams.py#L240). If this change fixes the build, I'm afraid it is false positive

Comment (lazebnyi) [reply airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml:41]: I'm not sure because cursors for incremental tests generated here - https://github.com/airbytehq/airbyte/blob/787b2c33d7586c9a3401dd72e4fdbc2c4ccbf80c/airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_incremental.py#L61

Comment (davydov-d) [reply airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml:41]: That's right. It falls back to default value (what is in `configured_catalog_incremental.json`) if you dont set it in the acceptance-test-config.yml manually

File: airbyte-integrations/connectors/source-sentry/acceptance-test-config.yml [hunk 2]
```diff
@@ -8,6 +8,19 @@ acceptance_tests:
...
+          bypass_reason: "Order features return randomly"
+        - name: organization/features
+          bypass_reason: "Order features return randomly"
+        - name: plugins/*/features
+          bypass_reason: "Order features return randomly"
+        projects:
+          - name: features
+            bypass_reason: "Order features return randomly"
+          - name: organization/features
+            bypass_reason: "Order features return randomly"
```
Comment (evantahler) [inline airbyte-integrations/connectors/source-sentry/acceptance-test-config.yml:23]: For the random ordering-type problems.  If it isn't a crazy ask, I think it would be better to sort the record properties ourself rather than skip testing that stream or property.

Comment (lazebnyi) [reply airbyte-integrations/connectors/source-sentry/acceptance-test-config.yml:23]: I was create follow up issue https://github.com/airbytehq/airbyte/issues/23860 to unblock this PR.
